spel/minimal-linux.json

```diff
@@ -571,6 +572,7 @@
         "spel_amigen8manfst": "",
         "spel_amigen8source": "https://github.com/plus3it/AMIgen8.git",
         "spel_amigen8storlay": "",
+        "spel_amigen8rootnm": "",
```

Comment:
This needs to be "sorted" in the map keys so it is alphabetical

Reply:
one would think that as the parent of a toddler I would be better at the alphabet....